app/lib/pre_assembly/copy_stager.rb

```diff
@@ -6,6 +6,20 @@ class CopyStager
     def self.stage(source, destination)
       FileUtils.mkdir_p File.dirname(destination)
       FileUtils.cp_r source, destination
+      recursively_chmod_based_on_type!(destination)
+    end
+
+    def self.recursively_chmod_based_on_type!(path)
+      unless File.directory?(path)
+        FileUtils.chmod 0o0664, path
+        return
+      end
+
+      FileUtils.chmod 0o0775, path
+
+      Dir["#{path}/*"].each do |f|
+        recursively_chmod_based_on_type!(f)
+      end
```

Comment:
These lines show as uncovered. Given this and my other comment below, I guess the question for @sul-dlss/infrastructure-team is... does `CopyStager` ever get a `destination` that is a directory? Or is it always a file? If the latter, that'd simplify this PR a bit and bump the coverage back up.

Reply:
uh - wouldn't hierarchical files imply we do have directories?

Reply:
It's hard to know the answer to this definitively.   I would suspect the answer is we will never get a directory, as we don't accession or process empty directories to my knowledge... we only process files, and while files can be in sub-directories now (the hierarchical work we did), this would still be a file copy operation (which would presumably create any folders that contain that file as part of the file copy operation).

Reply:
I suppose if we really wanted to find out, we can pull it out and put in one of those HB style "hey, this thing happened!" kind of alerts

Reply:
The reason why I ask is the prior line has `cp_r` which implies the expectation of directories (in which case `cp` should suffice). I like the idea of throwing in an HB alert! I'll do that.

Reply:
Added the HB experiment in.